DotNet/.github/workflows/stale.yml

```diff
@@ -0,0 +1,30 @@
+name: "Close stale issues"
+
+# Controls when the workflow will run. 
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  # Cron job time seems to be UTC  
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # stale:
+  stale-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: "This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. If you need additional assistance please contact Esri Technical Support. Thank you for your contributions."
+          close-issue-message: "This issue has been automatically closed due to inactivity. If you need additional assistance please contact Esri Technical Support."
+          stale-pr-message: "This PR has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. If you need additional assistance please contact Esri Technical Support. Thank you for your contributions."
+          close-pr-message: "This PR has been automatically closed due to inactivity. If you need additional assistance please contact Esri Technical Support."          
+          only-labels: null
```

Comment:
If the default value is "unset" for `only-labels` should we remove this entry line?

Reply:
Yes, good catch!